app/src/pages/OnDeviceDisplay/ProtocolDashboard/ProtocolRow.tsx

```diff
@@ -0,0 +1,210 @@
+      const runId: string = data.data.id
+      history.push(`/protocols/${runId}/setup`)
+    },
+  })
+
+  const handlePinnedProtocolIds = (
+    pinnedProtocolIds: PinnedProtocolIds
+  ): void => {
+    dispatch(
+      updateConfigValue('protocols.pinnedProtocolIds', pinnedProtocolIds)
+    )
+  }
+
+  // TODO const { deleteProtocol } = useDeleteProtocolMutation(protocol.id)
+
+  const handleDeleteClick = (): void => {
+    longpress.setIsLongPressed(false)
+    // TODO: deleteProtocol()
+    console.log(`deleted protocol with id ${protocol.id}`)
+    history.go(0)
+  }
+
+  const handlePinClick = (): void => {
+    longpress.setIsLongPressed(false)
+    if (pinned) {
+      pinnedProtocolIds = pinnedProtocolIds.filter(p => p !== protocol.id)
+    } else {
+      pinnedProtocolIds.push(protocol.id)
+    }
+    handlePinnedProtocolIds(pinnedProtocolIds)
+  }
+
+  const handleProtocolClick = (): void => {
+    if (!longpress.isLongPressed) {
+      history.push(`/protocols/${protocol.id}`)
+    }
+  }
+
+  const handleRunClick = (): void => {
+    longpress.setIsLongPressed(false)
+    createRun({ protocolId: protocol.id })
+  }
+
+  return (
+    <TableRow
+      key={protocol.key}
+      onClick={() => handleProtocolClick()}
+      ref={longpress.ref}
+    >
+      <TableDatum>
+        <StyledText
+          fontSize="1.5rem"
+          lineHeight="2.0625rem"
+          fontWeight={TYPOGRAPHY.fontWeightSemiBold}
+        >
+          {truncateString(protocolName, 88, 66)}
+        </StyledText>
+      </TableDatum>
+      <TableDatum>
+        <StyledText
+          fontSize="1.375rem"
+          lineHeight="1.75rem"
+          fontWeight={TYPOGRAPHY.fontWeightRegular}
+        >
+          {lastRun != null
+            ? formatDistance(new Date(lastRun), new Date(), {
+                addSuffix: true,
+              })
+            : t('no_history')}
+        </StyledText>
+      </TableDatum>
+      <TableDatum>
+        <StyledText
+          fontSize="1.375rem"
+          lineHeight="1.75rem"
+          fontWeight={TYPOGRAPHY.fontWeightRegular}
+        >
+          {format(new Date(protocol.createdAt), 'Pp')}
+        </StyledText>
+        {longpress.isLongPressed && (
+          <ModalShell width="15.625rem">
+            <Flex
+              flexDirection={DIRECTION_COLUMN}
+              justifyContent={JUSTIFY_CENTER}
+            >
+              <Flex
+                alignItems={ALIGN_CENTER}
+                gridGap={SPACING.spacing3}
+                height="4.875rem"
+                justifyContent={JUSTIFY_CENTER}
+                padding={SPACING.spacing5}
+                onClick={() => handleRunClick()}
+              >
+                <Icon name="play-circle" size="1.5rem" color={COLORS.black} />
+                <StyledText
+                  fontSize="1.375rem"
+                  lineHeight="1.5rem"
+                  fontWeight={TYPOGRAPHY.fontWeightRegular}
+                  textAlign={TEXT_ALIGN_CENTER}
```

Comment:
`TEXT_ALIGN_CENTER` is an old const and still we haven't switched all of them.

```suggestion
                  textAlign={TYPOGRAPHY.textAlignCenter}
```